<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Test-Jpwin11" failures="9" tests="80" name="test.script.RenamePersonalFolderTest" time="1007.696" errors="1" timestamp="2024-09-17T14:15:19 JST" skipped="1">
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithKataganaSinlgeByte" time="11.927">
    <failure message="expected [ｶｻﾀﾅﾊﾏﾔﾗﾜｱｶｻﾀﾅﾊﾏﾔﾗﾜｱｶｻﾀﾅﾊﾏﾔﾗﾜｱｶｻﾀﾅﾊﾏﾔﾗﾜｱｶｻﾀﾅﾊﾏﾔﾗﾜｱｶｻﾀﾅﾊﾏﾔﾗﾜﾗﾊﾏﾔﾗﾜﾗﾜｱｶﾊﾏﾔﾗﾜﾗｻﾀﾅﾊﾊ] but found [AutoTest]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [ｶｻﾀﾅﾊﾏﾔﾗﾜｱｶｻﾀﾅﾊﾏﾔﾗﾜｱｶｻﾀﾅﾊﾏﾔﾗﾜｱｶｻﾀﾅﾊﾏﾔﾗﾜｱｶｻﾀﾅﾊﾏﾔﾗﾜｱｶｻﾀﾅﾊﾏﾔﾗﾜﾗﾊﾏﾔﾗﾜﾗﾜｱｶﾊﾏﾔﾗﾜﾗｻﾀﾅﾊﾊ] but found [AutoTest]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at test.script.RenamePersonalFolderTest.verifyFileIsRenameWithKataganaSinlgeByte(RenamePersonalFolderTest.java:1145)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyFileIsRenameWithKataganaSinlgeByte -->
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithCON" time="12.853"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithKataganaSinlgeByte85chars" time="13.001"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithLPT3" time="13.035"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameEndWithDot" time="13.033"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithEnglishSinlgeByte" time="12.052">
    <failure message="expected [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZIJKLMNOPQRSTUVWXYVWXYZIJKLMN] but found [AutoTest]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZIJKLMNOPQRSTUVWXYVWXYZIJKLMN] but found [AutoTest]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at test.script.RenamePersonalFolderTest.verifyFileIsRenameWithEnglishSinlgeByte(RenamePersonalFolderTest.java:1098)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyFileIsRenameWithEnglishSinlgeByte -->
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithKanjiMultiByte85chars" time="13.206"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithLPT2" time="12.933"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithCOM1" time="12.952"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithlpt9" time="13.242"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithcom2" time="12.936"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithBackSlash" time="12.923"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithEnglishMultiByte" time="12.021">
    <failure message="expected [ａｂｃｄｅｆｇｈｉｊｋｌｍｎｏｐｑｒｓｔｕｖｗｘｙｚＡＢＣＤＥＦＧＨＩＪＫＬＭＮＯＰＱＲＳＴＵＶＷＸＹＺＰＱＲＳＴＵＶＷＸＹＺＴＵＶＷＸＹＴＵＶＷＸＹＺＰＱＲＳ] but found [AutoTest]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [ａｂｃｄｅｆｇｈｉｊｋｌｍｎｏｐｑｒｓｔｕｖｗｘｙｚＡＢＣＤＥＦＧＨＩＪＫＬＭＮＯＰＱＲＳＴＵＶＷＸＹＺＰＱＲＳＴＵＶＷＸＹＺＴＵＶＷＸＹＴＵＶＷＸＹＺＰＱＲＳ] but found [AutoTest]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at test.script.RenamePersonalFolderTest.verifyFileIsRenameWithEnglishMultiByte(RenamePersonalFolderTest.java:1109)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyFileIsRenameWithEnglishMultiByte -->
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithclock$" time="13.056"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithcom9" time="13.041"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithKataganaMultiByte85chars" time="13.121"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithcom4" time="13.078"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithCOM6" time="13.021"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithCOM9" time="12.813"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithNumberMultiByte" time="12.173">
    <failure message="expected [１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１] but found [AutoTest]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１１] but found [AutoTest]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at test.script.RenamePersonalFolderTest.verifyFileIsRenameWithNumberMultiByte(RenamePersonalFolderTest.java:1133)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyFileIsRenameWithNumberMultiByte -->
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithNumberSinlgeByte85chars" time="13.177"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithEnglishMultiByte85chars" time="13.176"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithlpt3" time="12.943"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithNumberMultiByte85chars" time="13.188"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithHiraganaMultiByte" time="12.178">
    <failure message="expected [あぁかさたないぃきしちにあぁかさたないぃきしちにあぁかさたないぃきしちにあぁかぁかさたないぃきしちにあぁかさたないぃきしちにあさたないぃきしちにあぁかさたないぃ] but found [AutoTest]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [あぁかさたないぃきしちにあぁかさたないぃきしちにあぁかさたないぃきしちにあぁかぁかさたないぃきしちにあぁかさたないぃきしちにあさたないぃきしちにあぁかさたないぃ] but found [AutoTest]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at test.script.RenamePersonalFolderTest.verifyFileIsRenameWithHiraganaMultiByte(RenamePersonalFolderTest.java:1168)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyFileIsRenameWithHiraganaMultiByte -->
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithEnglishSinlgeByte85chars" time="13.218"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithSNAPSHOT" time="12.951"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithcom5" time="13.083"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithLPT9" time="12.878"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithlpt6" time="12.903"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithlpt5" time="12.949"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithLPT7" time="13.213"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithLPT5" time="13.108"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithKanjiMultiByte" time="12.393">
    <failure message="expected [教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教教育漢字教育漢字教育漢] but found [AutoTest]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教育漢字教教育漢字教育漢字教育漢] but found [AutoTest]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at test.script.RenamePersonalFolderTest.verifyFileIsRenameWithKanjiMultiByte(RenamePersonalFolderTest.java:1180)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyFileIsRenameWithKanjiMultiByte -->
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithCLOCK$" time="12.863"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithAUX" time="13.283"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithlpt0" time="13.198"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithnul" time="12.955"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithCOM4" time="13.253"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithlpt7" time="12.722"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithcom1" time="13.095"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithcom7" time="13.294"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithSnapshot" time="13.254"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithLPT4" time="13.142"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithSpecialcharsMultiByte" time="12.118">
    <failure message="expected [－｀～［｛］～｝＇；．，！＠＃＄％＾＆（）－｀～［｛］～｝＇；．，！＠＃＄％＿＋＝－｀～［｛］｝＇；．，！＠＃＄％＾＆（）＿＋＝－｀～［｛］｝＇；．，！＠＃＄＋] but found [AutoTest]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [－｀～［｛］～｝＇；．，！＠＃＄％＾＆（）－｀～［｛］～｝＇；．，！＠＃＄％＿＋＝－｀～［｛］｝＇；．，！＠＃＄％＾＆（）＿＋＝－｀～［｛］｝＇；．，！＠＃＄＋] but found [AutoTest]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at test.script.RenamePersonalFolderTest.verifyFileIsRenameWithSpecialcharsMultiByte(RenamePersonalFolderTest.java:1192)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyFileIsRenameWithSpecialcharsMultiByte -->
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithCOM0" time="13.060"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithCOM2" time="12.920"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameStartWithSpace" time="13.596"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithForwardSlash" time="12.999"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithaux" time="12.887"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithsnapshot" time="12.981"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithKataganaMultiByte" time="12.146">
    <failure message="expected [アカサタナハマヤラワアカサタナハマヤラワアカサタナハマヤラワアカサタナハマヤラワアカサタナハマヤラワアカサタナハマヤヤラワアカサタナハマヤラワアカサタナハマヤヤ] but found [AutoTest]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [アカサタナハマヤラワアカサタナハマヤラワアカサタナハマヤラワアカサタナハマヤラワアカサタナハマヤラワアカサタナハマヤヤラワアカサタナハマヤラワアカサタナハマヤヤ] but found [AutoTest]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at test.script.RenamePersonalFolderTest.verifyFileIsRenameWithKataganaMultiByte(RenamePersonalFolderTest.java:1156)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyFileIsRenameWithKataganaMultiByte -->
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithcom0" time="13.005"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithcom6" time="13.018"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithlpt2" time="13.030"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithLPT1" time="12.883"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithlpt1" time="13.225"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithNumberSinlgeByte" time="12.045">
    <failure message="expected [11111111111111111111111111111111111111111111111111111111111111111111111111111111] but found [AutoTest]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [11111111111111111111111111111111111111111111111111111111111111111111111111111111] but found [AutoTest]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at test.script.RenamePersonalFolderTest.verifyFileIsRenameWithNumberSinlgeByte(RenamePersonalFolderTest.java:1121)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyFileIsRenameWithNumberSinlgeByte -->
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithLPT8" time="13.158"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithSNAPSHOTS" time="13.518"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithlpt4" time="13.241"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithCOM8" time="12.865"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithCOM5" time="13.209"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithPRN" time="13.219"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithlpt8" time="12.716"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithCOM7" time="13.324"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithSpecialcharsMultiByte85chars" time="13.257"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenamebutton" time="8.615"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithcom3" time="13.117"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithCOM3" time="13.008"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithsnapshots" time="13.061"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithcom8" time="13.039"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithHiraganaMultiByte85chars" time="13.168"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithLPT0" time="12.979"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="closeBrowser" time="7.270">
    <error message="Error communicating with the remote browser. It may have died.
Build info: version: &#039;unknown&#039;, revision: &#039;unknown&#039;, time: &#039;unknown&#039;
System info: host: &#039;TEST-JPWIN11&#039;, ip: &#039;192.168.81.68&#039;, os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;20.0.1&#039;
Driver info: driver.version: RemoteWebDriver
Capabilities [{networkConnectionEnabled=false, chrome={chromedriverVersion=128.0.6613.86 (3045ed680fdab64f784f3b366cfe74adbdf6ff01-refs/branch-heads/6613@{#1337}), userDataDir=C:\Windows\SystemTemp\scoped_dir18016_1573970684}, timeouts={implicit=0, pageLoad=300000, script=30000}, pageLoadStrategy=normal, unhandledPromptBehavior=dismiss and notify, strictFileInteractability=false, platform=XP, proxy=Proxy(), webauthn:extension:minPinLength=true, goog:chromeOptions={debuggerAddress=localhost:51127}, webauthn:extension:credBlob=true, webauthn:extension:prf=true, acceptInsecureCerts=false, browserVersion=128.0.6613.120, fedcm:accounts=true, browserName=chrome, javascriptEnabled=true, platformName=XP, setWindowRect=true, webauthn:extension:largeBlob=true, webauthn:virtualAuthenticators=true}]
Session ID: 053b85bbd84d30c45df73db248f1421d" type="org.openqa.selenium.remote.UnreachableBrowserException">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'TEST-JPWIN11', ip: '192.168.81.68', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '20.0.1'
Driver info: driver.version: RemoteWebDriver
Capabilities [{networkConnectionEnabled=false, chrome={chromedriverVersion=128.0.6613.86 (3045ed680fdab64f784f3b366cfe74adbdf6ff01-refs/branch-heads/6613@{#1337}), userDataDir=C:\Windows\SystemTemp\scoped_dir18016_1573970684}, timeouts={implicit=0, pageLoad=300000, script=30000}, pageLoadStrategy=normal, unhandledPromptBehavior=dismiss and notify, strictFileInteractability=false, platform=XP, proxy=Proxy(), webauthn:extension:minPinLength=true, goog:chromeOptions={debuggerAddress=localhost:51127}, webauthn:extension:credBlob=true, webauthn:extension:prf=true, acceptInsecureCerts=false, browserVersion=128.0.6613.120, fedcm:accounts=true, browserName=chrome, javascriptEnabled=true, platformName=XP, setWindowRect=true, webauthn:extension:largeBlob=true, webauthn:virtualAuthenticators=true}]
Session ID: 053b85bbd84d30c45df73db248f1421d
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:667)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:703)
at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:542)
at cocoro.tests.BaseTest.closeBrowser(BaseTest.java:88)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.internal.invokers.TestMethodWorker.invokeAfterClassConfigurations(TestMethodWorker.java:229)
at org.testng.internal.invokers.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:215)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:131)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.RuntimeException: Process refused to die after 10 seconds, and couldn't taskkill it: Couldn't detect pid
at org.openqa.selenium.os.ProcessUtils.killWinProcess(ProcessUtils.java:133)
at org.openqa.selenium.os.ProcessUtils.killProcess(ProcessUtils.java:81)
at org.openqa.selenium.os.UnixProcess$SeleniumWatchDog.destroyHarder(UnixProcess.java:265)
at org.openqa.selenium.os.UnixProcess$SeleniumWatchDog.access$200(UnixProcess.java:219)
at org.openqa.selenium.os.UnixProcess.destroy(UnixProcess.java:127)
at org.openqa.selenium.os.CommandLine.destroy(CommandLine.java:153)
at org.openqa.selenium.remote.service.DriverService.stop(DriverService.java:220)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:94)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:646)
... 28 more
Caused by: java.lang.RuntimeException: Couldn't detect pid
at org.openqa.selenium.os.ProcessUtils.getProcessId(ProcessUtils.java:195)
at org.openqa.selenium.os.ProcessUtils.killWinProcess(ProcessUtils.java:128)
... 36 more
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private final long java.lang.ProcessImpl.handle accessible: module java.base does not "opens java.lang" to unnamed module @78e03bb5
at java.base/java.lang.reflect.AccessibleObject.throwInaccessibleObjectException(AccessibleObject.java:387)
at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:363)
at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:311)
at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:181)
at java.base/java.lang.reflect.Field.setAccessible(Field.java:175)
at org.openqa.selenium.os.ProcessUtils.getProcessId(ProcessUtils.java:181)
... 37 more
]]>
    </error>
  </testcase> <!-- closeBrowser -->
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithNUL" time="13.171"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithLPT6" time="13.032"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithcon" time="12.989"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameWithprn" time="6.993"/>
  <testcase classname="test.script.RenamePersonalFolderTest" name="verifyFileIsRenameEndWithSpace">
    <skipped/>
  </testcase> <!-- verifyFileIsRenameEndWithSpace -->
</testsuite> <!-- test.script.RenamePersonalFolderTest -->
